crates/networking/rpc/engine/payload.rs

```diff
@@ -783,7 +783,10 @@ fn validate_fork(block: &Block, fork: Fork, context: &RpcApiContext) -> Result<(
 }
 
 async fn get_payload(payload_id: u64, context: &RpcApiContext) -> Result<PayloadBundle, RpcErr> {
-    debug!("Requested payload with id: {:#018x}", payload_id);
+    info!(
+        id = format!("{:#018x}", payload_id),
```

Comment:
Should we format the ID in the same way in the "Fork choice updated includes payload attributes" message?